cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -205,7 +205,7 @@ func (s *CMServer) Run(_ []string) error {
 	// file, and then overriding the Master flag, if non-empty.
 	kubeconfig, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 		&clientcmd.ClientConfigLoadingRules{ExplicitPath: s.Kubeconfig},
-		&clientcmd.ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: s.Master}}).ClientConfig()
+		&clientcmd.ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: s.Master}}).ClientConfig("")
```

Comment:
So this is temporary but going in? Could the final solution be done in this PR, or is it a lot of work? If it's too much work, is there some external issue tracking the remaining change? Where?

Thanks :smile: